src/collective/liveblog/browser/templates/microupdate.pt

```diff
@@ -0,0 +1,98 @@
+          itemtype="http://schema.org/BlogPosting">
+        <header>
+          <div tal:replace="structure provider:plone.abovecontenttitle" />
+          <h1 class="documentFirstHeading"
+              itemprop="headline"
+              tal:content="context/Title">
+          Title
+        </h1>
+          <div tal:replace="structure provider:plone.belowcontenttitle" />
+          <div class="documentDescription"
+              itemprop="description"
+              tal:content="context/Description">
+            Description
+          </div>
+        </header>
+        <div tal:replace="structure provider:plone.abovecontentbody" />
+        <div id="content-core"
+            itemprop="articleBody">
+          <section id="micro-updates">
+            <article class="microupdate"
+                data-timestamp=""
+                itemprop="comment"
+                itemscope
+                itemtype="http://schema.org/Comment"
+                tal:define="update view/update"
+                tal:condition="update"
+                tal:attributes="data-timestamp update/timestamp">
+              <h2 class="microupdate-title"
+                  itemprop="headline"
+                  tal:condition="update/title"
+                  tal:content="update/title" />
+              <div class="microupdate-text"
+                  itemprop="text"
+                  tal:content="structure update/text" />
+              <div class="microupdate-byline">
+                <p tal:condition="view/show_byline"
+                    i18n:translate="">
+                  By
+                  <span itemprop="author"
+                      tal:content="update/creator"
+                      i18n:name="byline" />
+                </p>
+                <time itemprop="datePublished"
+                    title="2014-08-28 14h31"
+                    datetime="2014-08-28T14:31:02.402"
+                    tal:attributes="datetime update/isoformat;
+                                    title update/datetime;
+                                    data-date update/date;
+                                    data-time update/time">
+                  <span class="microupdate-time"
+                      tal:content="update/time" />
+                </time>
+              </div>
+            </article>
+            <div class="microupdate"
+                data-timestamp=""
+                tal:condition="not:view/has_updates"
+                tal:attributes="data-timestamp view/now">
+              <p class="microupdate-title">
+                <strong i18n:translate="">No micro-updates yet for this Liveblog.</strong>
+              </p>
+            </div>
+          </section>
+        </div>
+        <div tal:replace="structure provider:plone.belowcontentbody" />
+          <div class="back"><span i18n:translate="" tal:omit-tag="">Extracted from:</span> <a tal:attributes="href context/absolute_url" tal:content="context/Title" /></div>
+      </article>
+      <script>
```

Comment:
ok, when I move the date / time into byline I'll remove it! thank you for the code review!